buddyboss/buddyboss-platform
# PROD-720 Fixed #3059 Reset the video player on theatre model close

### All Submissions:






### Changes proposed in this Pull Request:

Instead to assigning the video src="" (which was causing the issue),  used reset() to reset the video player

Fixes #3059.

### How to test the changes in this Pull Request:

1. On videos page goto videos and click to open it in fullscreen popup and close and see the error in console
2. Try the same on the timeline/activity feed, open the video in popup and close and open the picture

### Proof Screenshots or Video

https://somup.com/crl2620HKJ

### Other information:




### Changelog entry

> Used reset() to reset the video player
